### DISCUSSION TITLE
Abstract polytopes

### DISCUSSION THREAD

**Discussion by mecejide** [gap-system/gap#5459] [General]
Do any GAP packages provide functionality for working with abstract polytopes?

**Comment by fingolfin** [comment]
I am not really aware of any. There is the `polymaking` package which provides a very limited interface to polymake.

What kind of functionality do you need?

You may wish to checkout the [OSCAR project](https://www.oscar-system.org) which among other things contains GAP and Polymake and functionality combining both.

**Comment by mecejide** [comment]
Well, there will be soon; someone going by “Sycamore916” is making one.
